### PR TITLE
translate beginner_source/chatbot_tutorial

### PR DESCRIPTION
**Pull request by Jo-ad-fontes**
## 라이선스 동의
_변경해주시는 내용에 BSD 3항 라이선스가 적용됨을 동의해주셔야 합니다._<br />
_더 자세한 내용은 [기여하기 문서](https://github.com/9bow/PyTorch-tutorials-kr/blob/master/CONTRIBUTING.md)를 참고해주세요._<br />
_동의하시면 아래 `[ ]`를 `[x]`로 만들어주세요._<br />

- [x] 기여하기 문서를 확인하였으며, 본 PR 내용에 BSD 3항 라이선스가 적용됨에 동의합니다.

## 관련 이슈 번호
_이 Pull Request와 관련있는 이슈 번호를 적어주세요._<br />
_이슈 또는 PR 번호 앞에 #을 붙이시면 제목을 바로 확인하실 수 있습니다. (예. #999 )_

- **이슈 번호**: #(숫자)

## PR 종류
_이 PR에 해당되는 종류 앞의 `[ ]`을 `[x]`로 변경해주세요._<br />
- [ ] 오탈자를 수정하거나 번역을 개선하는 기여
- [x] 번역되지 않은 튜토리얼을 번역하는 기여
- [ ] 공식 튜토리얼 내용을 반영하는 기여
- [ ] 위 종류에 포함되지 않는 기여

## PR 설명
_이 PR로 무엇이 달라지는지 대략적으로 알려주세요._
- beginner_source/chatbot_tutorial 앞 부분을 번역하였습니다.

### CODE REVIEW

File: beginner_source/text_sentiment_ngrams_tutorial.py [hunk 1]
```diff
@@ -1,20 +1,21 @@
...
 
-In this tutorial, we will show how to use the torchtext library to build the dataset for the text classification analysis. Users will have the flexibility to
+이 튜토리얼에서는 torchtext 라이브러리를 사용하여 텍스트 분류 분석을 위한 데이터셋을 구축하는 방법을 보여줍니다. 
+사용자는 다음과 같이 유동적으로 기능을 사용할 수 있습니다.
 
-   - Access to the raw data as an iterator
-   - Build data processing pipeline to convert the raw text strings into ``torch.Tensor`` that can be used to train the model
-   - Shuffle and iterate the data with `torch.utils.data.DataLoader <https://pytorch.org/docs/stable/data.html?highlight=dataloader#torch.utils.data.DataLoader>`__
+   - 반복자로 원시 데이터에 액세스합니다.
+   - 원시 텍스트 문자열을 모델 학습에 사용할 수 있는 ``torch.Tensor``로 변환하는 데이터 처리 파이프라인 구축합니다.
```
Comment (jafffy) [inline beginner_source/text_sentiment_ngrams_tutorial.py:9]: Typo로 보이는데, "데이터 처리 파이프라인 구축합니다." -> "데이터 처리 파이프라인을 구축합니다." 로 변경하시면 좋을 것 같습니다.

File: beginner_source/text_sentiment_ngrams_tutorial.py [hunk 2]
```diff
@@ -1,20 +1,21 @@
...
+   - torch.utils.data.DataLodaer로 데이터를 섞거나 반복합니다. `torch.utils.data.DataLoader <https://pytorch.org/docs/stable/data.html?highlight=dataloader#torch.utils.data.DataLoader>`__
```
Comment (jafffy) [inline beginner_source/text_sentiment_ngrams_tutorial.py:10]: "섞거나" 가 mix인지 shuffle 인지 독자가 헷갈릴 수 있을 것 같고, 자칫 DataLoader가 데이터를 섞는 것처럼 오독할 가능성이 있어, 괄호하고 shuffle을 써주거나 무작위 추출 (이 번역어가 최적인지는 잘 모르겠습니다만...) 로 변경하시면 어떨까 싶습니다.

**Review by jafffy — CHANGES_REQUESTED** [review]

File: beginner_source/text_sentiment_ngrams_tutorial.py [hunk 3]
```diff
@@ -1,20 +1,21 @@
...
 """
 
 
 ######################################################################
-# Access to the raw dataset iterators
+# 원시 데이터셋 반복자들을 액세스하기
 # -----------------------------------
 #
-# The torchtext library provides a few raw dataset iterators, which yield the raw text strings. For example, the ``AG_NEWS`` dataset iterators yield the raw data as a tuple of label and text.
+# torchtext 라이브러리는 원시 텍스트 문자열을 생성하는 몇 가지 원시 데이터셋 반복자를 제공합니다. 예를 들어 ``AG_NEWS`` 데이터셋 반복자는 원시 데이터를 레이블과 텍스트의 튜플로 생성합니다.
```
Comment (hrxorxm) [inline beginner_source/text_sentiment_ngrams_tutorial.py:18]: 두 문장을 두 줄로 작성하시면 가로 스크롤 없이 보기 편할 것 같습니다!

File: beginner_source/text_sentiment_ngrams_tutorial.py [hunk 4]
```diff
@@ -25,33 +26,30 @@
...
+#     방위 산업에서 시기 적절하고 때때로 논란이 되는 연극을 제작하는 것으로 
+#     평판이 좋은 민간 투자 회사인 칼라일 그룹은 조용히 시장의 다른 부분에 베팅했습니다.')
 # 
 #     next(train_iter)
-#     >>> (3, "Oil and Economy Cloud Stocks' Outlook (Reuters) Reuters - Soaring 
-#     crude prices plus worries\\about the economy and the outlook for earnings are 
-#     expected to\\hang over the stock market next week during the depth of 
-#     the\\summer doldrums.")
+#     >>> (3, "석유 및 경제 클라우드 주식 전망 (Reuters) Reuters - 
+#     유가 급등과 경제에 대한 우려, 실적 전망이 여름 침체기인 다음 주 주식 시장을 뒤흔들 것으로 예상됩니다.")
```
Comment (hrxorxm) [inline beginner_source/text_sentiment_ngrams_tutorial.py:39]: 이 부분은 데이터셋을 확인하는 부분 같은데 데이터셋 안에 있는 영어들은 번역할 필요가 없지 않을까 합니다!

File: beginner_source/text_sentiment_ngrams_tutorial.py [hunk 5]
```diff
@@ -25,33 +26,30 @@
...
 #
 
 
 ######################################################################
-# Prepare data processing pipelines
+# 데이터 처리 파이프 라인 준비하기
 # ---------------------------------
 #
-# We have revisited the very basic components of the torchtext library, including vocab, word vectors, tokenizer. Those are the basic data processing building blocks for raw text string.
+# 우리는 어휘, 단어 벡터, 토크나이저를 포함하여 torchtext 라이브러리의 가장 기본적인 구성 요소를 다시 방문했습니다. 이러한 요소는 원시 텍스트 문자열에 대한 기본 데이터 처리 빌딩 블록입니다.
```
Comment (hyoyoung) [inline beginner_source/text_sentiment_ngrams_tutorial.py:47]: 주어는 빼주시고
revisit는 여기서 다시 확인했다는 의미이므로 의역을 해주시는게 어떨까 합니다

**Review by hyoyoung — CHANGES_REQUESTED** [review]

File: beginner_source/text_sentiment_ngrams_tutorial.py (hunk 1 above)
Comment (Jo-ad-fontes) [reply beginner_source/text_sentiment_ngrams_tutorial.py:9]: 이전 번역에서 '섞다'로 쓴 걸 봐서 적용하긴 했는데 확실히 괄호 추가가 필요해 보이네요. 추천해주신 무작위 추출같이 좀 더 고민해보면 좋은 표현이 있을 것 같기도 합니다.

File: beginner_source/text_sentiment_ngrams_tutorial.py (hunk 2 above)
Comment (Jo-ad-fontes) [reply beginner_source/text_sentiment_ngrams_tutorial.py:10]: 이전 번역에서 '섞다'로 쓴 걸 봐서 적용하긴 했는데 확실히 괄호 추가가 필요해 보이네요. 추천해주신 무작위 추출같이 좀 더 고민해보면 좋은 표현이 있을 것 같기도 합니다.

File: beginner_source/text_sentiment_ngrams_tutorial.py (hunk 3 above)
Comment (Jo-ad-fontes) [reply beginner_source/text_sentiment_ngrams_tutorial.py:18]: 넵 수정하겠습니다!

File: beginner_source/text_sentiment_ngrams_tutorial.py (hunk 5 above)
Comment (Jo-ad-fontes) [reply beginner_source/text_sentiment_ngrams_tutorial.py]: 좋은 의견 감사합니다
